src/components/App.js

```diff
@@ -140,6 +112,38 @@ export const App = ({ defaultSource, defaultConfig, defCustomPlugin, defaultId,
   importDefaultPlugins();
   registerDefaultPlugins();
 
+  useEffect(() => {
+
+    setPanes(jsonConfig.map((config, index) => {
+
+      return {
+        menuItem: 'Config ' + index, render: () =>
+          <><Output
```

Comment:
What does the `<>` do?

Reply:
It's a "react fragment" shorthand:
https://reactjs.org/docs/fragments.html#short-syntax